gtsfm/utils/metrics.py

```diff
@@ -0,0 +1,195 @@
+"""Utilities to compute and save evaluation metrics.
+
+Authors: Akshay Krishnan
+"""
+import json
+import numpy as np
+import os
+from dask.delayed import Delayed
+from typing import Dict, List, Optional, Tuple, Union
+
+from gtsam import Point3, Rot3, Pose3, Unit3
+
+import gtsfm.utils.geometry_comparisons as comp_utils
+from gtsfm.common.sfm_result import SfmResult
+
+# A StatsDict is a dict from string to optional floats or their lists.
+StatsDict = Dict[str, Union[Optional[float], List[Optional[float]]]]
+
+
+def get_errors_statistics(errors: List[Optional[float]]) -> StatsDict:
```

Comment:
I'd rename it to `compute_error_summary` or `compute_errors_statistics` as it is not a getter.

Reply:
`compute_errors_statistics` sounds good.